File: Sort/insertion/insertion.c
void insertion(int* datas, int size)
{
    int cur;
    for(int i = 1 ; i < size ; i++)
    {
        cur = datas[i];
        int j = i - 1;
        while(j >= 0 && datas[j] > cur)
        {
            datas[j+1] = datas[j];
            j--;
        }
        datas[j+1] = cur;
    }
    return;
}

File: Sort/insertion/insertion.py
def insertion(datas):
    size = len(datas)
    for i in range(1,size):
        cur = datas[i]
        j = i-1
        while j >= 0 and cur < datas[j]:
            datas[j+1] = datas[j]
            j = j-1
        datas[j+1]=cur
